src/services/modifiedLinesCollector/ModifiedMethod.groovy

```diff
@@ -0,0 +1,40 @@
+package services.modifiedLinesCollector
+
+class ModifiedMethod {
+
+    private String signature;
+    private Set<ModifiedLine> lines;
```

Comment:
please add a comment here, and possibly rename, explaining the meaning of lines. is this all lines in a method? is this all modified lines? what's the meaning of this for methods in the merge commit?